src/hotspot/share/opto/superword.cpp

```diff
@@ -2665,7 +2611,6 @@ bool SuperWord::output() {
       }
 #endif
 
-      _block.at_put(i, vn);
```

Comment:
Note: this is useless. Plus, it would be nice to have the `VLoopBody` as read-only in SuperWord.